src/app/modules/main/profile_section/keycard/module.nim

```diff
@@ -176,13 +184,22 @@ proc buildKeycardItem(self: Module, walletAccounts: seq[WalletAccountDto], keyPa
         return accounts[i]
     return nil
 
+  let isAccountInKnownAccounts = proc(knownAccounts: seq[WalletAccountDto], address: string): bool =
+    for i in 0 ..< knownAccounts.len:
+      if(knownAccounts[i].address == address):
+        return true
+    return false
+
```

Comment:
minor, you can use `any` here from `sequtils`
Example:
let numbers = @[1, 4, 5, 8, 9, 7, 4]
assert any(numbers, proc (x: int): bool = x > 8) == true
assert any(numbers, proc (x: int): bool = x > 9) == false